### DISCUSSION TITLE
How to call a stored procedure with mixed parameters: parameter list contains int and bytea

### DISCUSSION THREAD

**Discussion by wuruilongll** [PostgREST/postgrest#2722] [Q&A] [answered]
### Environment

* PostgreSQL version: 13
* PostgREST version: 9.0
* Operating system: Linux

### Description of issue

I have written a function as follows：
```sql
create or replace function public.img_cache_server(x int, y int,img bytea)
returns void
language plpgsql
as 
$$
	begin
		insert into img_base_cache values(x,y,img);
	end;
$$
```

when i call it  by set the    'Content-Type': 'application/octet-stream' as  follows, it can not call normally, how to set the request headers or  body parameters?

```javascript
     let parameters = {}
      parameters['x'] = 0
      parameters['y'] = 0
      //data is the img bytes get from remote address
      parameters['img'] = data
      fetch('http://localhost:3000/rpc/img_cache_server',
            {
              headers: {
                'Content-Type': 'application/octet-stream',
                //'Content-Type': 'application/json'
              },
              method: 'post', 
              body: parameters
            }
          ).then(function(response) {

          });
```

**Comment by steve-chavez** [comment, marked as answer]
As a workaround, you could send `x` and `y` on headers and get them through https://postgrest.org/en/stable/api.html#accessing-request-headers-cookies-and-jwt-claims.

To support this use case cleanly, we would need https://github.com/PostgREST/postgrest/issues/922
